lib/disasters.js

```diff
@@ -145,7 +163,82 @@ function getDisaster(disasterId, callback) {
   });
 }
 
+async.auto({
+  disasters: ['operations', cacheDisasters],
+},
+function (err, results) {
+  if (err) {
+    console.log("hid_profiles cron run failed.");
+  }
+  else {
+    console.log("Finished hid_profiles cron run successfully.");
+  }
+  process.exit(err ? 1 : 0);
+});
+
 
+function newDisasterNotification(disaster) {
+  var requiredLocations = [];  
+  async.series([
+    function (cb) {
+      if(disaster.countries.length > 0){
+        async.forEachSeries(disaster.countries, function(country, callback2) {
+          if (!requiredLocations[iso3Countries[country]]) {
+                requiredLocations[iso3Countries[country]] = {};
+                requiredLocations[iso3Countries[country]].glide = [];
+                requiredLocations[iso3Countries[country]].contacts = [];
+                requiredLocations[iso3Countries[country]].disasterName = [];
+              }  
+            requiredLocations[iso3Countries[country]].glide.push(disaster.glide_id);
+            Contact.find({ $query: { type: 'local', location: iso3Countries[country]},  $orderby : {location: -1} }, function (err, contact){
+              contact.forEach(function(contact){
+                if(!contact){
+                  console.log("No users found");
+                  return cb();
+                }   
+                requiredLocations[iso3Countries[country]].contacts.push(contact);       
+            });   
+            callback2();         
+          })
+        }, cb);
+      }
+    },
+    function (cb){
+      var mailOptions = {};
+      mailOptions.requiredLocations = requiredLocations;
+      mailOptions.baseUrl = process.env.APP_BASE_URL;
+      mailOptions.subject = 'New Disaster in your region';
+      mailOptions.contacts = false;
+
+      if(disaster.countries.length > 0){
+        async.forEachSeries(disaster.countries, function(country, cb3) {
+          if(requiredLocations[iso3Countries[country]].contacts.length > 0)
+            mailOptions.contacts = true;
+
+          if (mailOptions.contacts === true) {
+            
+            requiredLocations[iso3Countries[country]].contacts.forEach(function(contact){
+                mailOptions.to = contact.email[0].address;
+                mailOptions.nameGiven = contact.nameGiven;
+                mailOptions.nameFamily = contact.nameFamily;
+                mailOptions.id = contact._id;
+                mailOptions.location = contact.location;
+                mail.sendTemplate('new_disaster_notification', mailOptions, function (err, info) {
+                          if (err) {
+                    console.log("ERR", err);
+                  } 
+                });
+            })
+          }
+          else {
+            cb3();
```

Comment:
oh sorry, it is supposed to call cb3() after the mail template has been sent and there is no error.